src/Controller/Controller.php

```diff
@@ -106,7 +106,7 @@ class Controller implements EventListenerInterface, EventDispatcherInterface
      *
      * @var string
      */
-    protected $name;
+    protected string $name = '';
```

Comment:
This is to avoid a `?string` type because psalm complains when you check if initialized with `isset()`.